portfolio/src/main/webapp/index.html

```diff
@@ -8,26 +8,24 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">    
     <script src="script.js"></script>
   </head>
-  <body>
-      <div class = "header">
+  
+  <body id = "body">
+      <div id = "header">
            <a href="gallery.html"><i class="material-icons">collections</i></a>  
            <a href="http://www.linkedin.com/in/yareth-m-melendez-rivera" target="_blank"><i class="fa fa-linkedin-square" style="font-size:36px"></i> </a> 
            <a href="https://github.com/yarethmelendez" target="_blank"><i class="fa fa-github" style="font-size:36px"></i></a>
-
+           <h3> <a href = "spsTasks.html"> SPS-Tasks </a> <a href = "form.html"> Contact Me </a></h3>
```

Comment:
```suggestion
           <h3>
             <a href = "spsTasks.html"> SPS-Tasks </a>
             <a href = "form.html"> Contact Me </a>
           </h3>
```

Use new lines for new elements.